src/canvas/Canvas.ts

```diff
@@ -1101,15 +1101,14 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
           pointer = this.getScenePoint(e),
           mouseDownHandler =
             control && control.getMouseDownHandler(e, target, control);
-        if (mouseDownHandler) {
+        mouseDownHandler &&
           mouseDownHandler.call(
             control,
             e,
             this._currentTransform!,
             pointer.x,
             pointer.y
           );
-        }
```

Comment:
those changes here are applied by the minifier later on, but imho they take away on readibility. In the default safe lint or prettier profile are not considered, so i won't hold anything on it since may fall in the opinion realm